src/components/Progress/Progress.stories.tsx

```diff
@@ -0,0 +1,346 @@
+import React from 'react';
+import Progress from './Progress';
+import useSimulateProgress from '../../utils/hooks/useSimulateProgress';
+
+export default {
+  title: 'Progress',
+  component: Progress,
+};
+
+export const DefaultProgress = () => {
+  const progress = useSimulateProgress();
+
+  return <Progress pgValue={progress} />;
+};
+
+export const DiffTypeProgress = () => {
+  const progress = useSimulateProgress();
+
+  return (
+    <div>
+      <h1>Linear</h1>
+      <br />
+      <Progress pgValue={progress} />
+      <br />
+
+      <h1>Circular</h1>
+      <br />
+      <Progress pgType="circular" pgValue={progress} />
+      <br />
+    </div>
+  );
+};
+
+export const DiffSizeProgress = () => {
+  const progress = useSimulateProgress();
+
+  return (
+    <div>
+      <h1>Linear</h1>
+      <br />
+
+      <h3>xs</h3>
+      <Progress pgSize="xs" pgValue={progress} />
+      <br />
+
+      <h3>sm</h3>
+      <Progress pgSize="sm" pgValue={progress} />
+      <br />
+
+      <h3>default (no need to add props)</h3>
```

Comment:
make the default size more clear to user

Reply:
done